script/bundle-linux

```diff
@@ -39,7 +39,6 @@ version_info=$(rustc --version --verbose)
 host_line=$(echo "$version_info" | grep host)
 target_triple=${host_line#*: }
 musl_triple=${target_triple%-gnu}-musl
-remote_server_triple=${REMOTE_SERVER_TARGET:-"${musl_triple}"}
```

Comment:
Let's revert these changes to our linux bundling :)

Reply:
Whoops, accidentally left that in. Will remove for sure.